### PR TITLE
Add vue-paginate w/ npm auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10695 
@cdnjs/intern3 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/TahaSh/vue-paginate
 * NPM package url (optional):https://www.npmjs.com/package/vue-paginate
 * License and its reference: [MIT](https://github.com/TahaSh/vue-paginate#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 9
   - Count of stars: 228
   - Count of forks: 28
 * NPM download stats (optional):
   - Downloads in the last day: 90
   - Downloads in the last week: 355
   - Downloads in the last month: 1,382

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/vue-paginate/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "vue"
+  ],
+  "author": "Taha Shashtari",
+  "license": "MIT",
+  "npmName": "vue-paginate",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.gz)"
```
Comment (sufuf3) [inline ajax/libs/vue-paginate/package.json]: I think it's not a good way to use `**/!(*.gz)`
How about `**/*.+(js|css|map)`
What do you think?

**Comment by extend1994** [comment]
@hare1039 I think what you wanted to say was 1.0.0~1.2.2?
@sufuf3 I think your rule is a better choice so I modifies the rule.
As for map files, because there is no map files in latest version and we need to keep the consistency of the file structure. Namely, we don't want map file so I use the command `--no-map`.
Please help me review it, thanks.

File: ajax/libs/vue-paginate/package.json (hunk 1 above)
Comment (hare1039) [inline ajax/libs/vue-paginate/package.json]: I think  in folders like `dist`, release` etc... we should use a exclusive rule , not a inclusive rule.

Comment (clairetsai818) [inline ajax/libs/vue-paginate/package.json]: Is that always a better choice to us the inclusive rule? My thoughts were similar to @hare1039's.

**Comment by extend1994** [comment]
@PeterDaveHello Please help me review this pull request, thank you.

File: ajax/libs/vue-paginate/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "vue"
+  ],
+  "author": "Taha Shashtari",
+  "license": "MIT",
+  "npmName": "vue-paginate",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*.+(js|css|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: Use blacklist instead of whitelist in `dist` please.

Comment (extend1994) [inline ajax/libs/vue-paginate/package.json]: @PeterDaveHello Do you mean `**/*`

Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: I mean you should exclude to gzip file instead of select what file types should be added.

Comment (extend1994) [inline ajax/libs/vue-paginate/package.json]: @PeterDaveHello So `**/!(*.gz)` is a better choice?

Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: yep

**Comment by extend1994** [comment]
@PeterDaveHello I modified the rule, please help me review it, thank you.

File: ajax/libs/vue-paginate/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: @sufuf3 think about why you'll ask a perfect answer to a wrong one, as this has been mentioned not only once, and as you can see all your mates have the correct concept in this part.

Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: https://github.com/cdnjs/cdnjs/pull/10956#pullrequestreview-30489475

Comment (PeterDaveHello) [inline ajax/libs/vue-paginate/package.json]: @extend1994 I think you should discuss first before take any action especially in this case looks like two of your mates have the same thought with you.

**Pull request merged**